integration-tests/src/test/resources/features/filter.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Filter tests
+
+  Background:
+    Given authenticate against Manager
+    And create a new Bridge "mybridge"
+    And the Bridge "mybridge" is existing with status "ready" within 4 minutes
+    And the Ingress of Bridge "mybridge" is available within 2 minutes
+    And add a Processor to the Bridge "mybridge" with body:
+    """
+    {
+  "name": "myProcessor",
+  "action": {
+    "parameters": {"topic":  "demoTopic"},
+    "type": "KafkaTopic"
+  },
+  "filters": [
+    {
+      "key": "source",
+      "type": "StringEquals",
+      "value": "StorageService"
+    }
+  ]
+}
+    """
```

Comment:
Please adjust alignment, see for example https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/integration-tests/src/test/resources/features/action-webhook.feature#L11-L22